src/coord/info_command.c

```diff
@@ -7,6 +7,7 @@
 #include "info_command.h"
 #include "resp3.h"
 #include "info/field_spec_info.h"
+#include "../src/reply_macros.h"
```

Comment:
This mixes our regular code with the coordinator, is that a practice we are okay with?